include/networkit/viz/Point.hpp

```diff
@@ -30,18 +30,18 @@ namespace NetworKit {
  *
  * Points in any dimension of templated type.
  */
-template<class T>
+template<class T = coord>
 class Point {
```

Comment:
Should this class be moved to the toplevel NetworKit namespace to avoid importing the viz module from other modules?

Reply:
I don't think it's necessary, since it's a header-only dependency and as such can be hidden from CMake.